_posts/2023-10-21-beneath-the-shell.md

```diff
@@ -0,0 +1,65 @@
+
+As the Fetri'sei began splitting between the ones reaching empty systems and the ones destroyed before replicating once, the central government started looking around for options; the Ka'mar, once touted to be on their way to become a juggernaut, had completely and publicly failed to deliver on the automation project, only completing a few subscale demonstrators and small surface facilities. Wether this was caused by sabotage or simple mismanagement is irrelevant; what counts is that this option was now unviable, making the choice of a super-sized organism controlled warship obvious. Still anxious about the power of the Ka'sei, a new department was created for the purpose: the Ka'het, integrating the power systems and limited automation created by the Ka'mar over the years with the super-sizing technology.
+
+## The Ka'het, and three-way strategy
+
+As more details were being defined about the Ka'het and their function, a number of military doctrines started to be attached to the concept; for example, the entirely automated construction of the warships was selected on the basis of the Assured Continued Production, intended to keep the manufacturing lines open even in the case of a war with heavy population losses that made it impossible to find enough qualified personnel.
+Whereas most war fleets are are composed by a large amount different vehicles, making sure to fill out every niche, it was decided early on in the development of the Ka'het architecture that this wasn't desiderable here; the Ka'mar automated manufacturing technology was often either massive and reliable or small and slow, sometimes even unpredictable. Making a factory capable of building a wide variety of vehicles would have been absurdly expensive, so the focus was put on three, highly specific vehicles: the Maeri, the Telis and the Vareti.
+
+### 1: The Maeri'het
+
+<img class="centered" src="/images/blog/ka-het/maeri-het-exploded.jpg" width="570" height="370" />
+
+The smallest of the Ka'het vehicles is also the most unique. Developed as a long range scout, it has the capability of completely (but temporarily) shielding itself from all kinds of sensors, the workings of which are kept secret from everyone not directly involved with the project. Although its future use hasn't been specified, it's plausible to think they will follow the footsteps of the Fetri, travelling sublight to enemy systems outside the main cluster and destroying any foe trying to pass through - or perhaps, once the Celeatis research group succeeds in its mission, it could be the the first ship to pass through the collapsed anomaly. While away from a control ship the Maeri is mostly autonomous, deciding the best path forward without external imput; however, it can also be given orders from its supervisors, as to ensure it will not interfer where it's not supposed to.
```

Comment:
```suggestion
The smallest of the Ka'het vehicles is also the most unique. Developed as a long range scout, it has the capability of completely (but temporarily) shielding itself from all kinds of sensors, the workings of which are kept secret from everyone not directly involved with the project. Although its future use hasn't been specified, it's plausible to think they will follow the footsteps of the Fetri, traveling sublight to enemy systems outside the main cluster and destroying any foe trying to pass through - or perhaps, once the Celeatis research group succeeds in its mission, it could be the the first ship to pass through the collapsed anomaly. While away from a control ship the Maeri is mostly autonomous, deciding the best path forward without external imput; however, it can also be given orders from its supervisors, as to ensure it will not interfere where it's not supposed to.
```